UnityProject/Assets/Scripts/Objects/Canisters/Canister.cs

```diff
@@ -145,31 +144,6 @@ private void OnDisable()
 			UpdateManager.Remove(CallbackType.PERIODIC_UPDATE, UpdateMe);
 		}
 
-		//this is just here so anyone trying to change the armor value in inspector sees it being
-		//reset
-		private void OnValidate()
-		{
-			SetDefaultIntegrity();
-		}
-
-		private void SetDefaultIntegrity()
```

Comment:
lol what was this?

Reply:
idk should never need a automatically setting Armor